packages/react/src/FilteredActionList/FilteredActionListWithDeprecatedActionList.tsx

```diff
@@ -40,14 +48,18 @@ const StyledHeader = styled.div`
 
 export function FilteredActionList({
   loading = false,
+  loadingType = FilteredActionListLoadingTypes.bodySpinner,
   placeholderText,
   filterValue: externalFilterValue,
   onFilterChange,
+  onListContainerRefChanged,
+  onInputRefChanged,
   items,
   textInputProps,
   inputRef: providedInputRef,
   sx,
   className,
+  announcementsEnabled: _announcementsEnabled = true,
```

Comment:
are we adding this here just so the APIs match? I see no use in this file. If so that's fine, just checking

Reply:
Yep, you got it 👍 Hopefully this will all go away in the near future when we get rid of `FilteredActionListWithDeprecatedActionList`.